src/main/java/by/exadel/internship/team_four/entity/Form.java

```diff
@@ -0,0 +1,42 @@
+package by.exadel.internship.team_four.entity;
```

Comment:
Достаточно названия приложения
```suggestion
package by.exadel.internship.entity;
```

Reply:
done